src/modules/ekf2/EKF/ev_height_control.cpp

```diff
@@ -90,6 +90,11 @@ void Ekf::controlEvHeightFusion(const extVisionSample &ev_sample, const bool com
 					   math::max(_params.ev_pos_innov_gate, 1.f),
 					   aid_src);
 
+	// filtered innovation for preflight checks
+	if (!aid_src.innovation_rejected) {
```

Comment:
The issue with this is that if the measurement is continuously rejected, the lpf won't update and the preflight check will pass. I would rather prefer if we update the lpf all the time; however, the potential issue is that it could then fail on a spike.

Reply:
Ideally shouldn't we have it lead back to the state machine and then control mode timing out? 

If anything I start to have slightly larger questions about what we're hoping to achieve, checking per aid source innovations vs velocity or position innovations or variances generally. Then it also goes back to actual operating requirements, what's optional vs required, things you have have preflight that might not work until in the air (distance sensor and optical flow too close to the ground). Anyway, maybe we should talk this through briefly...